skiko/buildSrc/src/main/kotlin/fileExtUtils.kt

```diff
@@ -0,0 +1,7 @@
+val OS.dynamicLibExt: String
+    get() = when (this) {
+        OS.Linux -> ".so"
+        OS.Windows -> ".dll"
+        OS.MacOS -> ".dylib"
+        OS.Wasm, OS.IOS -> error("Should not be called for $this")
```

Comment:
Why not support IOS? Shall be .dylib as well. And in case of Wasm -> it's `.wasm`.